screens/WalletDetailScreen.tsx

```diff
@@ -197,6 +235,34 @@ const WalletDetailScreen = ({
       ? null
       : amountDecimal;
 
+  type tokenProps = { tokenId: string };
+
+  const HideButton = ({ tokenId }: tokenProps) => (
+    <VisibilityArea>
+      <VisibilityText
+        onPress={() => {
+          addTokenToBlacklist(tokenId);
+        }}
+      >
+        Visible in Vault
```

Comment:
Actions should be `verbs` which describe what will happen when pressed, but after looking at this and experimenting a bit, I think this may look best without any text at all.    
If it's confusing, then we could tie in the toggle to the available messaging box on the screen.  But that could happen later, for now I would try without any text and see if that looks good.

I do like the location here for this button, not pressing it at by mistake, even on my small phone.